src/test/kotlin/interactor/GetCityHasCheapestInternetConnectionInteractorTest.kt

```diff
@@ -0,0 +1,76 @@
+package interactor
+
+import dataSource.FakeDataSource
+import model.CityEntity
+import org.junit.jupiter.api.Test
+import org.junit.jupiter.api.Assertions.*
+import org.junit.jupiter.api.BeforeEach
+import org.junit.jupiter.api.TestInstance
+import org.junit.jupiter.api.function.Executable
+
+
+@TestInstance(TestInstance.Lifecycle.PER_CLASS)
+internal class GetCityHasCheapestInternetConnectionInteractorTest {
+
+    private lateinit var cityHasCheapestInternet: GetCityHasCheapestInternetConnectionInteractor
+    private lateinit var fakeDataSource: FakeDataSource
+
+    @BeforeEach
+    fun setup(){
+        fakeDataSource = FakeDataSource()
+        cityHasCheapestInternet = GetCityHasCheapestInternetConnectionInteractor()
+    }
+
+
+
+    @Test
+    fun should_ReturnTheOnlyCity_When_GivenOnly1CityInHighQualityData(){
```

Comment:
It will be more efficient to use this test naming convention:
```suggestion
    fun `should return the only city when given only 1 city in high quality data`(){
```

Reply:
I think i named it like bareq in the testing course